docs/pages/versions/unversioned/sdk/apple-authentication.mdx

```diff
@@ -8,10 +8,14 @@ import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/Con
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
+import { SnackInline } from '~/ui/components/Snippet';
 
 `expo-apple-authentication` provides Apple authentication for iOS 13+. It does not yet support lower iOS versions, Android, or the web.
 
-Beginning with iOS 13, any app that includes third-party authentication options **must** provide Apple authentication as an option in order to comply with App Store Review guidelines. Learn more about Apple authentication on the ["Sign In with Apple" website](https://developer.apple.com/sign-in-with-apple/).
+Beginning with iOS 13, any app that includes third-party authentication options **must** provide Apple authentication as an option
+in order to comply with App Store Review guidelines.
+
+Learn more about Apple authentication on the ["Sign In with Apple" website](https://developer.apple.com/sign-in-with-apple/).
```

Comment:
```suggestion
Beginning with iOS 13, any app that includes third-party authentication options **must** provide Apple authentication as an option
in order to comply with App Store Review guidelines. For more information, see Apple authentication on the [Sign In with Apple](https://developer.apple.com/sign-in-with-apple/) website.
```

I think we can join the sentences here and change the verbiage for consistency reasons. At other places, we started to use `For more information...` like context.